tasks/support/templates/appliance_installation_script.rb.erb

```diff
@@ -36,34 +39,33 @@ VMDB_DIR                 = "/var/www/miq/vmdb"
 #
 #     https://github.com/pypa/get-pip/blob/master/tasks/generate.py
 #
-MIQPERF_GEM_BUNDLE = <<-GEM
-<%= @gem_base64_string -%>
-GEM
+<% unless @solo_gem -%>
+<%= render_partial "gem_base64_string", :env_name => "MIQPERF", :gem_base64_string => @gem_base64_string -%>
+<% end -%>
 
 <% if @stackprof_gemspec and @stackprof_gem_base64_string -%>
-STACKPROF_GEM_BUNDLE = <<-STACKPROF_GEM
-<%= @stackprof_gem_base64_string -%>
-STACKPROF_GEM
+<%= render_partial "gem_base64_string", :env_name => "STACKPROF", :gem_base64_string => @stackprof_gem_base64_string -%>
 <% end -%>
 
+<% if @other_gems -%>
```

Comment:
one day - would be cool if all gems were treated the same.

Reply:
Agreed.  On the TODO list for sure, but for now, want to close this out.  :shipitkeenan: